templates/advantage/index.html

```diff
@@ -282,28 +284,32 @@ <h4>EAL2</h4>
 {% else %}
 <section class="p-strip">
   <div class="row u-equal-height">
-    <div class="col-6 u-vertically-center">
-      <div>
-        <h2>Already subscribed?</h2>
-        <p><a href="/login" class="p-button--positive">Sign in</a> to access your personal or paid subscriptions.</p>
-      </div>
+    <div class="col-12 u-vertically-center">
+      <h2>Already subscribed?</h2>
+      <p class="u-no-margin--bottom"><a href="/login" class="p-button--positive">Sign in</a> to access your personal or paid subscriptions.</p>
     </div>
-    <div class="col-6">
-      <div class="p-card">
-        <h3>Free for personal use</h3>
-        <p>Anyone can use UA Infrastructure Essential for free on up to 3 machines (limitations apply).  All you need is an Ubuntu One account. And if you’re a recognised <a href="https://wiki.ubuntu.com/Membership" class="p-link--external">Ubuntu community member</a>, it’s free on up to 50 machines.</p>
-        <p class="u-no-margin--bottom"><a href="/login" class="p-button--neutral">Register</a> to get your free subscription.</p>
-      </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <h2>Free for personal use</h2>
+      <p>Anyone can use UA Infrastructure Essential for free on up to 3 machines (limitations apply).  All you need is an Ubuntu One account. And if you’re a recognised <a href="https://wiki.ubuntu.com/Membership" class="p-link--external">Ubuntu community member</a>, it’s free on up to 50 machines.</p>
```

Comment:
–1: This contradicts the static text above, which says “You are entitled to use Ubuntu Advantage Essential on 3 machines” whether you’re an Ubuntu community member or not.

Either the community entitlement is included in this release, in which case that text should query whether you’re a community member and show “3” or “50” as appropriate.

Or the community entitlement is _not_ included in this release, in which case the “And if you’re a recognised Ubuntu community member…” sentence here should be omitted for now.

Reply:
We don't know if they are a member so I will remove the text.